doc/release-notes/rl-2511.section.md

```diff
@@ -14,6 +14,10 @@
 
 <!-- To avoid merge conflicts, consider adding your item at an arbitrary place in the list instead. -->
 
+- `nixVersions.nix_2_3` has been dropped because it was insecure and unmaintained.
```

Comment:
Should this also mention `nixVersions.minimum`, which has also been removed?

On the one hand, aliases don't usually need to be mentioned in release notes. On the other hand, this alias is more like a meta package that isn't tightly coupled to 2.3.


```suggestion
- `nixVersions.nix_2_3` has been dropped because it was insecure and unmaintained.
  Its `nixVersions.minimum` alias has also been dropped for the same reason.
```


```suggestion
- `nixVersions.nix_2_3` and `nixVersions.minimum` have been dropped because they were insecure and unmaintained.
```

?